charts/nfdiv-case-api/Chart.yaml

```diff
@@ -15,3 +15,7 @@ dependencies:
     repository: '@hmctspublic'
     tags:
         - ccd-idam-pr
+  - name: xui-webapp
+    version: ~0.2.1
+    repository: '@hmctspublic'
+    condition: xui-webapp.enabled
```

Comment:
should this value be set to false in values yaml ?

Reply:
yes

Reply:
just checked you will need to set default to false in values yaml and bump chart version